react/src/AddDatasets/components/DataEntryTable.tsx

```diff
@@ -377,6 +387,12 @@ function DataEntryToolbar(props: {
                 <Box display="flex" flexGrow={1}>
                     <Typography variant="h6">Enter Metadata</Typography>
                 </Box>
+                <GroupDropdownSelect
+                    selectedGroupCodes={props.groups}
+                    allGroupCodes={props.allGroups}
+                    onChange={props.setGroups}
+                    disabled={props.allGroups.length <= 1}
```

Comment:
When I first sign in I always get an error on this line that `allGroups` is undefined when navigating to add datasets. This is fixed by a hard refresh.